pep-0541.txt

```diff
@@ -109,6 +109,7 @@ following means of contact:
 
 The maintainers stop trying to reach the user after six weeks.
 
+Ownership change requests are to be raised at [9]_
```

Comment:
This should be moved under "Removal of an abandoned project" instead. Please keep two newlines at the end of the section.